docs/modules/ROOT/pages/explanation/kyverno-policies.adoc

```diff
@@ -0,0 +1,7 @@
+= Kyverno policies used by {product}
+
+As documented in xref:appuio-cloud:ROOT:explanation/decisions/kyverno-policy.adoc[this decision] and xref:appuio-cloud:ROOT:explanation/decisions/kyverno-generator.adoc[this decision], {product} uses Kyverno as its policy engine and resource generator.
```

Comment:
I was once looking into accessible web design. There I learned that links with texts like _here_, _this x_ etc. are an issue for people with disabilities. The suggestion was to replace them with something more telling. I realized that this then makes it also easier to read for people without disabilities. At least it was for me.

Reply:
Yeah, I try to avoid using that style of links when possible, but I couldn't figure out a way to phrase it differently yesterday. I'll have a look again.

Reply:
I've rephrased this sentence, it should be much better now

Reply:
It most certainly is, thanks.